### PR TITLE
Prepare for wgsl-analyzer upgrades

### PR DESCRIPTION
**Pull request by DJMcNab**
Blocked on:
- https://github.com/wgsl-analyzer/wgsl-analyzer/pull/41

Nice to haves:
- https://github.com/wgsl-analyzer/wgsl-analyzer/pull/39
- https://github.com/wgsl-analyzer/wgsl-analyzer/pull/42
- https://github.com/wgsl-analyzer/wgsl-analyzer/pull/40

### CODE REVIEW

File: piet-wgsl/shader/path_coarse.wgsl [hunk 1]
```diff
@@ -295,19 +295,22 @@ fn main(
...
+                            let p = vec2<f32>(tile_x0, y_edge);
                             if dp.x < 0.0 {
                                 tile_seg.delta = p - lp0;
                             } else {
                                 tile_seg.origin = p;
                                 tile_seg.delta = lp1 - p;
                             }
                             if tile_seg.delta.x == 0.0 {
-                                tile_seg.delta.x = sign(dp.x) * 1e-9;
+                                tile_seg.delta.x = sign(dp.x) * 1
```
Comment (raphlinus) [inline piet-wgsl/shader/path_coarse.wgsl:306]: This doesn't seem right to me. Did you mean to write out the number in unscientific notation?

Comment (DJMcNab) [reply piet-wgsl/shader/path_coarse.wgsl:306]: This got auto-formatted by wgsl-analyzer. It shouldn't be here

**Review by raphlinus — APPROVED** [review]
As discussed on Zulip, this is still not working great for me, but I think it makes sense for this to go in (especially to get rid of the hard-coded paths) and I can rebase my work-in-progress on top of it.

**Comment by raphlinus** [comment]
Oops, this should have been against `dev` instead of `main`. I'll fix it up. This is a much bigger paper cut than I was hoping for :(

**Pull request merged**